<launch>
    <node pkg="decision_maker" type="decision_maker_node" name="decision_maker_node">
        <!-- input topic name -->
        <param name="cm2ext_topic_name" type="string" value="/hellocm/cm2ext"/>
        <param name="objects_topic_name" type="string" value="/object_detection/objects"/>

        <!-- output topic name -->
        <param name="ext2cm_topic_name" type="string" value="/hellocm/ext2cm"/>

        <!-- PID gains -->
        <param name="latitude_steer_kp" type="double" value="6.0"/>
        <param name="latitude_steer_ki" type="double" value="0.0"/>
        <param name="latitude_steer_kd" type="double" value="0.01"/>

        <param name="longitude_velocity_kp" type="double" value="0.5"/>
        <param name="longitude_velocity_ki" type="double" value="0.0"/>
        <param name="longitude_velocity_kd" type="double" value="0.0"/>

        <param name="longitude_distance_kp" type="double" value="0.12"/>
        <param name="longitude_distance_ki" type="double" value="0"/>
        <param name="longitude_distance_kd" type="double" value="0"/>

        <!-- tmp -->
        <param name="velocity_max_kph" type="double" value="40"/>
    </node>
</launch>